server/src/test/java/org/elasticsearch/monitor/jvm/JvmInfoTests.java

```diff
@@ -36,6 +37,15 @@ public void testUseG1GC() {
         }
     }
 
+    public void testDnsTtl() {
+        String propertyValue = System.getProperty("networkaddress.cache.ttl");
```

Comment:
As you mention in the comment... I think it is safe to grab the property here, set it to the the desired value, test it, then set it back to the original value in a finally block. Assuming we don't execute any of the unit tests within the same JVM concurrently (I don't think we do, but not 100%).